server/start.ts

```diff
@@ -17,6 +17,10 @@ import nouns from './indexers/nouns';
 import { getLatestAuction } from './db/queries';
 import balances from './indexers/balances';
 import transfers from './indexers/transfers';
+import { init } from '@airstack/node';
+import socials from './indexers/socials';
+
+init(process.env.AIRSTACK_API_KEY || '');
```

Comment:
this is not great — global state and could have race condition. Can we query airstack directly via fetch and avoid importing `airstack/node` module?